File: exams/MobileCars/app/src/main/java/com/bvd/android/carstobert/MainActivity.java
package com.bvd.android.carstobert;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.bvd.android.carstobert.customer.CustomerActivity;
import com.bvd.android.carstobert.employee.EmployeeActivity;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
    }
    @OnClick(R.id.mainCustomerBtn)
    void redirectCustomer() {
        Intent intent = new Intent(this, CustomerActivity.class);
        startActivity(intent);
    }
    @OnClick(R.id.mainEmployeBtn)
    void  redirectEmployee() {
        Intent intent = new Intent(this, EmployeeActivity.class);
        startActivity(intent);
    }
}

File: exams/at/lectures/3/HomeActivity/app/src/main/java/ro/cojocar/dan/homeactivity/MainActivity.kt
package ro.cojocar.dan.homeactivity

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.google.android.things.pio.PeripheralManager
import kotlinx.android.synthetic.main.activity_main.*

/**
 * Skeleton of an Android Things activity.
 *
 * Android Things peripheral APIs are accessible through the class
 * PeripheralManagerService. For example, the snippet below will open a GPIO pin and
 * set it to HIGH:
 *
 * <pre>{@code
 * val service = PeripheralManagerService()
 * val mLedGpio = service.openGpio("BCM6")
 * mLedGpio.setDirection(Gpio.DIRECTION_OUT_INITIALLY_LOW)
 * mLedGpio.value = true
 * }</pre>
 * <p>
 * For more complex peripherals, look for an existing user-space driver, or implement one if none
 * is available.
 *
 * @see <a href="https://github.com/androidthings/contrib-drivers#readme">https://github.com/androidthings/contrib-drivers#readme</a>
 *
 */
class MainActivity : Activity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_main)

    val manager = PeripheralManager.getInstance()
    logd("Available GPIO: ${manager.gpioList}")

    led.setOnClickListener {
      logd("Starting led activity")
      startActivity(Intent(this, BlinkActivity::class.java))
    }

    button.setOnClickListener {
      logd("Starting button activity")
      startActivity(Intent(this, ButtonActivity::class.java))
    }
  }
}
